src/api/app/controllers/webui/package_controller.rb

```diff
@@ -81,13 +81,16 @@ def show
     @comment = Comment.new
     @requests = []
     @services = Service.find(project: @project.name, package: @package.name)
+
+    switch_to_webui2
   end
 
   def main_object
     @package # used by mixins
   end
 
   def linking_packages
+    switch_to_webui2
```

Comment:
Why do you do that? You can surely delay the queries for `@package.linking_packages` in some other way. The same is true for all other dialogs.

Reply:
BTW not necessarily in this PR. I'm also happy with just recording a card in the backlog.